### DISCUSSION TITLE
Wanted to disable 2FA

### DISCUSSION THREAD

**Discussion by Maheshbr88** [community/community#48677] [New to GitHub]
I forgot my github account and i lost 2 FA recovery codes as well. Could you please guide me the procedure to dusable 2FA from my github account

**Comment by jge162** [comment]
You have to login to disable it.

**Comment by jge162** [comment]
Also when logging in it will tell you if you forgot your password, and 2fa info to fill out info and then github suport will review it to remove the 2fa.

**Comment by peterkuria1** [comment]
I am also facing the same issue on another important account.  I lost my two laptops and mobile phone in a house break-in at night. Unfortunately, I never expected that this could ever happen, and had backed the recovery codes on both laptops and nowhere in the cloud. Now I got a new phone and laptop but cannot disable 2FA. I am able to log in to my account using a password but cannot proceed beyond the MFA two-factor authentication. I did file a ticket on GitHub and it's now 4 days and going without hope of recovering my account. I hope Github can automate some of the verifications to speed up service delivery.
